README.md

```diff
@@ -23,6 +23,14 @@ Active Shipping is currently being used and improved in a production environment
 * [Kunaki](http://www.kunaki.com)
 * [Australia Post](http://auspost.com.au/)
 
+
+## Versions
+
+Development on `master` is for a pre-release of the `2.x` version of this gem. It includes a number of backwards incompatible changes and improvements. Progress can be tracked in [the milestone](https://github.com/Shopify/active_shipping/milestone/1).
+
+We are no longer actively developing version `1.x`. We have released `1.9` and will only backport small fixes to this version, on branch `1-9-stable`, and they should be on `master` first.
```

Comment:
I'm strongly opposed to this.

We want to encourage contributions, by us and by the OSS community, against the newest version. By default, look forward. We won't be unstable for long. And we can backport to `1-9-stable` in the cases where something is urgent.

This is also the way rails does it.